### PR TITLE
added instance segmentation task spaces

### PR DESCRIPTION
**Pull request by rashmibanthia**

### CODE REVIEW

File: tasks/src/image-segmentation/data.ts [hunk 1]
```diff
@@ -59,6 +59,10 @@ const taskData: TaskDataCustom = {
 			description: "A semantic segmentation application that predicts human silhouettes.",
 			id:          "keras-io/Human-Part-Segmentation",
 		},
+		{
+			description: "Cell Instance Segmentation - Sartorius Kaggle Competition",
```
Comment (merveenoyan) [inline tasks/src/image-segmentation/data.ts:63]: Can you give more simpler context on what this is about? You can take a look at the other definitions 🙂

**Review by merveenoyan — CHANGES_REQUESTED** [review]
Thanks a lot for this PR, I just left a comment, please take a look 🙂

**Comment by rashmibanthia** [comment]
Does this description sound any better or more complex :D ? 
“An instance segmentation application to predict neuronal cell types from microscopy images. (Sartorius Kaggle Competition)”

**Comment by merveenoyan** [comment]
@rashmibanthia maybe we could remove kaggle part and it would be less cryptic IMO 😅

File: tasks/src/image-segmentation/data.ts (hunk 1 above)
Comment (merveenoyan) [inline tasks/src/image-segmentation/data.ts:63]:
```suggestion
			description: "An instance segmentation application to predict neuronal cell types from microscopy images.",
```
Does this look good?

**Review by merveenoyan — COMMENTED** [review]
@rashmibanthia left suggestion 🙂

File: tasks/src/image-segmentation/data.ts (hunk 1 above)
Comment (rashmibanthia) [reply tasks/src/image-segmentation/data.ts]: Yes perfect, Thank you very much @merveenoyan.

**Pull request merged**